package.json

```diff
@@ -47,7 +52,7 @@
     "@react-native-community/eslint-plugin": "^1.1.0",
     "@types/jest": "^26.0.14",
     "@types/react": "^16.14.20",
-    "@types/react-native": "^0.66.0",
+    "@types/react-native": "^0.71.3",
```

Comment:
This would be a breaking change for customers running react native < `71.3`. It might be better for us to use `*` in this case so customers aren't forced to upgrade react native to get the latest SDK?

Reply:
It's in the `devDependencies` section so it is used only when developing things inside the library. I needed it to be able to include the types for the fabric spec without TS errors. It should not be visible to the end users at all.